tools/download_preset_fonts.R

```diff
@@ -15,7 +15,16 @@ extract_first_group <- function(x, pattern) {
 }
 
 download_and_copy_fonts <-  function(theme) {
-  theme_scss <- readLines(file.path(theme, "_bootswatch.scss"))
+  theme_file_choices <- paste0(c("_bootswatch", "_rules"), ".scss")
+  theme_files <- file.path(theme, theme_file_choices)
```

Comment:
Can the file choice be given as a variable?

```suggestion
download_and_copy_fonts <-  function(theme, rule_file = "_rules.scss") {
  theme_files <- file.path(theme, rule_file)
```

Reply:
Good call, thanks! Done in cfd03d29b6f06d14c4f5f69f1de10da31944b479